pwiz_tools/Shared/PanoramaClient/DirectoryPicker.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace pwiz.PanoramaClient
+{
+    public partial class DirectoryPicker : Form
+    {
+
+        private FolderBrowser folders;
+        public string State;
+        public string Selected;
+
+        public DirectoryPicker(Uri server, string user, string pass, bool showCheckBox, string state, bool showSkyFolders = false)
```

Comment:
server, user, and pass are not getting used so can be removed?

Reply:
I went ahead and removed the unused parameters and class variables that weren't being used.